### PR TITLE
Initial implementation of `bn perf`.

### PR DESCRIPTION
**Pull request by lavie**
Very few options so far, just how many requests and concurrency.
Tested manually, not yet automatically.

### CODE REVIEW

**Comment by lavie** [comment]
I think we should merge this even before automatic testing, since it will be useful for snap testing & demoing.

File: cli.js [hunk 1]
```diff
@@ -106,6 +106,38 @@ Usage: $0 <command> [options]` // eslint-disable-line comma-dangle
   }, async (argv) => {
     await handleCommand(argv, invokeHandler);
   })
+  .command('perf <function> [options]', 'Measure invocation latency (experimental)', (yargs0) => {
```
Comment (arielshaqed) [inline cli.js]: `yargs0`?  Is that a JS standard for this sort of usage?

File: lib/index.js [hunk 2]
```diff
@@ -132,6 +134,43 @@ const invokeHandler = async function invokeHandler(options) {
...
+  logger.info(`Running performance test on function ${options.function}.
+Executing ${options.maxRequests} invocations with ${options.concurrency} "thread${options.concurrency > 1 ? 's' : ''}".
+Stand by for results...
+`);
+  const report = await perf(options.function, options.maxRequests, options.concurrency);
+  logger.info('Perf summary');
+  logger.info('============');
+
+  logger.info(columnify({
+    'Total time': `${report.totalTimeSeconds.toFixed(1)} ms`,
```
Comment (arielshaqed) [inline lib/index.js]: I don't understand the units here: `totalTimeSeconds` is seconds, the suffix `ms` is milliseconds?

Comment (lavie) [reply lib/index.js]: :) fixed.

File: test/spec.yml [hunk 3]
```diff
@@ -12,6 +12,7 @@
                 bn deploy <function> [options]            Deploys a function to the cloud
                 bn remove <function> [options]            Remove a previously deployed function
                 bn invoke <function> [options]            Invoke a Binaris function
+                bn perf <function> [options]              Measure invocation latency (experimental)
```
Comment (michaeladda) [inline test/spec.yml]: the spec should also have an actual use of this command

**Comment by lavie** [comment]
@michaeladda please approve so it can be merged. already published to npm.

File: test/spec.yml (hunk 3 above)
Comment (lavie) [reply test/spec.yml]: #293

**Pull request merged**